content/Rust-1.87.0.md

```diff
@@ -0,0 +1,251 @@
++++
+path = "2025/05/15/Rust-1.87.0"
+title = "Announcing Rust 1.87.0"
+authors = ["The Rust Release Team"]
+aliases = [
+    "2025/05/15/Rust-1.87.0.html",
+    "releases/1.87.0",
+]
+
+[extra]
+release = true
++++
+
+The Rust team is happy to announce a new version of Rust, 1.87.0. Rust is a programming language empowering everyone to build reliable and efficient software.
```

Comment:
```suggestion
The Rust team is happy to announce a new version of Rust, 1.87.0. This release occurs on the ten-year anniversary of Rust 1.0, live from the stage at [RustWeek](https://rustweek.org/celebration/). Thank you to the myriad contributors who have worked on Rust, past and present. Here's to many more decades of Rust!

Rust is a programming language empowering everyone to build reliable and efficient software.
```

Reply:
Would be awesome to embed a photo of the celebration room filled with people directly into the blog post :)

Reply:
I think it'd also be good to make clear that the 1.87.0 release itself isn't a special release; just that the day is. There will probably be some press coverage, and I don't think they should report the specific features of 1.87.0 itself as the big 10 year milestone.

Happy to think of some text for this. Probably a separate paragraph at the top or something.